easybuild/easyblocks/q/quantumespresso.py

```diff
@@ -49,7 +49,6 @@ class EB_QuantumESPRESSO(ConfigureMake):
     def extra_options():
         """Custom easyconfig parameters for Quantum ESPRESSO."""
         extra_vars = {
-            'hybrid': [False, "Enable hybrid build (with OpenMP)", CUSTOM],
```

Comment:
You're right...but `hybrid = True` is broken for FFTW, it's a bit of a fluke that it works with MKL. Anyway the actual [performance is worse than with pure MPI](http://www.prace-ri.eu/IMG/pdf/enabling_ffte_library_and_fftw3_threading_in_quantum_espresso.pdf)